client/src/utils/PostCallUtil.ts

```diff
@@ -29,3 +32,9 @@ export const submitSurveyResponseUtil = async (
     //Add Error logging here;
   }
 };
+
+export const componentToShow = (renderPostCall: boolean, userRole: RoomParticipantRole): string => {
+  if (!renderPostCall && userRole === RoomParticipantRole.presenter) return PRESENTER;
```

Comment:
lets make this simpler, 
if(!renderPostCall) {
     userRole === RoomParticipantRole.presenter ? RESENTER : ATTENDEE
}
else return SURVEY

Reply:
Also i guess we wont need this with Ralph's changes, he has 
      <CallComposite
        adapter={callAdapter}
        fluentTheme={fluentTheme}
        formFactor={formFactorValue}
        callInvitationUrl={userRole === RoomParticipantRole.presenter ? roomsInfo.inviteParticipantUrl : undefined}
      />
      so we would just need anothe check in RoomsMeetingExperience for renderPostCall and we can get rid of this function